overlord/configstate/configcore/kernel.go

```diff
@@ -92,13 +96,9 @@ func validateCmdlineAppend(c RunTransaction) error {
 		}
 
 		logger.Debugf("validating %s=%q", opt, cmdAppend)
```

Comment:
I missed this in the previous PR, it probably needs a bit more context information